### PR TITLE
packaging fixes

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by IvanMalison** [comment]
Did you do this by hand or did you use something like pandoc?

**Comment by graingert** [comment]
Pandoc. PyPI doesn't support anything except rst

**Comment by IvanMalison** [comment]
yep. Just wanted to make sure it wasn't by hand.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```
Comment (IvanMalison) [inline setup.cfg]: What does this do?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+universal=1
```
Comment (IvanMalison) [inline setup.cfg]: Nice. So this creates a python2/3 compatible distribution?

Comment (graingert) [inline setup.cfg]: It allows you to create universal wheels when you run:

`python setup.py sdist bdist_wheel register upload`

**Pull request merged**